api/tests/routes/backoffice/accounts_test.py

```diff
@@ -515,8 +515,8 @@ class GetPublicAccountTest(GetEndpointHelper):
     endpoint_kwargs = {"user_id": 1}
     needed_permission = perm_models.Permissions.READ_PUBLIC_ACCOUNT
 
-    # session + current user + user data + bookings
-    expected_num_queries = 4
+    # session + current user + user data + bookings + BeneficiaryFraudCheck
+    expected_num_queries = 5
```

Comment:
On a déjà `sa.orm.joinedload(users_models.User.beneficiaryFraudChecks),` dans `render_public_account_details`, c'est donc une requête inutile de les charger une seconde fois.
```suggestion
    expected_num_queries = 5
```
```suggestion
    # session + current user + user data + bookings
    expected_num_queries = 4
```

Reply:
ligne 518 et 519 account test
```
    # session + current user + user data + bookings
    expected_num_queries = 4
```
FAIT